package.json

```diff
@@ -87,9 +89,12 @@
     "docs:build": "docpress build",
     "commit": "git-cz",
     "cover": "istanbul cover test/index.js",
-    "lint": "eslint bin/ test/ lib/",
+    "lint": "eslint bin/ test/ lib/ .scripts/",
     "test": "npm run lint && node test | tap-spec",
-    "posttest": "rimraf .tmp"
+    "posttest": "rimraf .tmp",
+    "preinstall": "node .scripts/try_rename cached_node_modules node_modules",
```

Comment:
I guess the docs say it is a bad idea because it can cause problems during installation but we skip the whole installation step, so I think we should be fine